specification/maps/data-plane/Search/preview/2023-04-01-preview/search.json

```diff
@@ -0,0 +1,1437 @@
+        "operationId": "Search_GetGeocodingBatch",
+        "x-ms-examples": {
+          "A Geocoding Batch API call containing 2 Geocoding queries": {
+            "$ref": "./examples/ForwardGeocodingBatch.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Accept-Language"
+          },
+          {
+            "name": "geocodingBatchRequestBody",
+            "in": "body",
+            "description": "The list of address geocoding queries/requests to process. The list can contain a max of 100 queries and must contain at least 1 query.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchRequestBody"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "207": {
+            "description": "Multi-Status. One or more batch items could not be processed and return different status code.",
+            "schema": {
+              "$ref": "#/definitions/GeocodingBatchResponse"
+            }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/2022-09-01-preview/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/search/polygon": {
+      "get": {
+        "description": "**Get Polygon**\n\n**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nSupplies polygon data of a geographical area outline such as a city or a country region.\n>[!Important]\n>By using this feature, you agree to the preview legal terms. See the [Preview Supplemental Terms](https://azure.microsoft.com/support/legal/preview-supplemental-terms/) for additional details.",
```

Comment:
The following will appear in the overview page in the description for this API, the reason is that the overview page does not support markdown

">[!Important]

For example:
![image](https://github.com/user-attachments/assets/b4bc5704-4c1c-48af-a30e-a4726fac452a)